tests/acceptance/00_basics/environment/proc-net.cf.sub

```diff
@@ -0,0 +1,38 @@
+# you can run this test directly with
+
+# CFENGINE_TEST_OVERRIDE_PROCDIR=`pwd`/00_basics/environment/proc-net.cf.proc testall 00_basics/environment/proc-net.cf.sub
+
+body common control
+{
+    inputs => { "../../default.cf.sub" };
+    bundlesequence => { default($(this.promise_filename)) };
+    version => "1.0";
+}
+
+bundle agent test
+{
+  vars:
+      "todo" slist => {
+                       "routes", "ipv6_routes",
+                       "ipv6_addresses",
+                       "proc_stats",
+                       "dev"
+      };
+
+      "$(todo)" data => mergedata("sys.$(todo)");
+      "default_gateway" string => "$(sys.default_gateway)";
+}
+
+bundle agent check
+{
+  vars:
+      "testname" string => regex_replace($(this.promise_filename), "\\.sub$", "", "");
+
+  methods:
+      "check"  usebundle => dcs_check_state(test,
+                                           "$(this.promise_filename).expected.json",
+                                           $(testname));
+      "check"  usebundle => dcs_check_state(test,
+                                           "$(this.promise_filename).expected.json",
+                                           $(this.promise_filename));
```

Comment:
Yes, it's intentional, so the test can be run standalone. I know it looks strange.